roles/zuul/files/jobs/ansible.yaml

```diff
@@ -0,0 +1,30 @@
+- builder:
+    name: install-ansible
+    builders:
+      - shell: |
+          tmpdir=$(mktemp -d)
+          trap "rm -rf $tmpdir" EXIT
+          virtualenv $tmpdir/venv
+          source $tmpdir/venv/bin/activate
+          pip install ansible
```

Comment:
i would think to use this as a builder you don't want to do the trapdir, but go for /opt/venvs/ansible and then symlink at least ansible, ansible-playbook and ansible-galaxy into /usr/local/bin or somewhere.